### PR TITLE
*: clean up some lint comments

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### What problem does this PR solve?

Ref #4322

### What is changed and how it works?

Remove some lint comments and make the code better.

### Check List

Tests

- Unit test

### Release note

```release-note
None
```

### CODE REVIEW

**Review by JmPotato — APPROVED** [review]

File: server/schedule/operator/operator_test.go [hunk 1]
```diff
@@ -338,13 +336,13 @@ func (s *testOperatorSuite) TestCheckExpired(c *C) {
 
 func (s *testOperatorSuite) TestCheck(c *C) {
 	{
-		region := s.newTestRegion(1, 1, [2]uint64{1, 1}, [2]uint64{2, 2})
+		region := s.newTestRegion(2, 2, [2]uint64{1, 1}, [2]uint64{2, 2})
```
Comment (lhy1024) [inline server/schedule/operator/operator_test.go:339]: why change it?

Comment (rleungx) [reply server/schedule/operator/operator_test.go:339]: To prevent lint error.

**Review by lhy1024 — APPROVED** [review]

**Pull request merged**